sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Other Changes
 
 - Added identity config validation to `ManagedIdentityCredential` to avoid non-deterministic states (e.g. both `resource_id` and `object_id` are specified). ([#36950](https://github.com/Azure/azure-sdk-for-python/pull/36950))
+- Added validation to `CloudShellCredential`. ([#36438](https://github.com/Azure/azure-sdk-for-python/issues/36438))
```

Comment:
Since, `CloudShellCredential` isn't a public API, we should refer to `ManagedIdentityCredential`.
```suggestion
- Additional validation was added for `ManagedIdentityCredential` in Azure Cloud Shell environments. ([#36438](https://github.com/Azure/azure-sdk-for-python/issues/36438))
```